src/app/language/language.component.spec.ts

```diff
@@ -33,4 +34,22 @@ describe('LanguageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should contain at least one langauge', () => {
+    expect(fixture.debugElement.nativeElement.querySelectorAll('option').length).toBeGreaterThan(1);
+  });
+
+  it('should contain english langauge', () => {
+    
+    function containtsOption(select, option) {
```

Comment:
IMO it's controversial but acceptable way of coding this, please consider adding types though.